packages/fx-core/resource/package.nls.json

```diff
@@ -507,7 +507,24 @@
   "core.common.NoServerInformation": "No server information is found in the OpenAPI description document.",
   "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
   "core.common.MissingOperationId": "Missing operationIds: %s.",
-  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI description document: only GET and POST methods are supported, additionally, there can be at most 5 required parameter, and no auth is allowed. \nFor more information visit: \"https://aka.ms/build-api-based-message-extension\".",
+  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI description document: only GET and POST methods are supported, additionally, there can be at most 5 required parameter, and no auth is allowed. \nFor more information visit: \"https://aka.ms/build-api-based-message-extension\". \nReasons for API incompatibility are listed below:\n%s",
```

Comment:
Proposed: No supported API found in the OpenAPI document. Only GET and POST methods are supported, with a maximum of 5 required parameters. Authentication is not allowed. \nFor more information visit: \"https://aka.ms/build-api-based-message-extension\". \nReasons for API incompatibility are listed below:\n%s"